File: fetch_live_kimchi.py
import requests

login_url = "https://billing.abpseeds.com/api/admin/auth/login"
products_url = "https://billing.abpseeds.com/api/admin/products"

# Log in
print("Logging in...")
login_res = requests.post(login_url, json={"email": "admin@company.com", "password": "Admin@123"})
token = login_res.json().get("token")

if token:
    headers = {"Authorization": f"Bearer {token}"}
    print("Fetching live products...")
    res = requests.get(products_url, headers=headers)
    if res.status_code == 200:
        products = res.json()
        kimchi = next((p for p in products if p.get("name").lower() == "kimchi"), None)
        if kimchi:
            print("FOUND KIMCHI PRODUCT:")
            print(f"ID: {kimchi.get('id')}")
            print(f"SKU: {kimchi.get('sku')}")
            print(f"Images field: {kimchi.get('images')}")
            print(f"IsActive: {kimchi.get('isActive')}")
        else:
            print("Product 'kimchi' not found in live products database.")
            print("Available products:", [p.get("name") for p in products])
    else:
        print(f"Failed to fetch products: {res.status_code} {res.text}")
else:
    print("Failed to authenticate.")

File: fetch_live_kimchi.js
const main = async () => {
  console.log("Logging in...");
  const loginRes = await fetch("https://billing.abpseeds.com/api/admin/auth/login", {
    method: "POST",
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify({ email: "admin@company.com", password: "Admin@123" })
  });
  const loginData = await loginRes.json();
  const token = loginData.token;

  if (token) {
    console.log("Fetching products...");
    const prodRes = await fetch("https://billing.abpseeds.com/api/admin/products", {
      headers: { "Authorization": `Bearer ${token}` }
    });
    const products = await prodRes.json();
    const kimchi = products.find(p => p.name.toLowerCase() === "kimchi");
    if (kimchi) {
      console.log("FOUND KIMCHI PRODUCT IN DB:");
      console.log(JSON.stringify(kimchi, null, 2));
    } else {
      console.log("Product 'kimchi' not found.");
      console.log("Available products:", products.map(p => p.name));
    }
  } else {
    console.log("Failed to log in.");
  }
};

main();
